fbw-common/src/wasm/systems/systems/src/navigation/adirs.rs

```diff
@@ -477,13 +477,42 @@ impl AdirsDiscreteOutputs for AirDataInertialReferenceSystem {
         self.adirus[adiru_number - 1].low_speed_warning_4_260kts()
     }
 }
+
 impl AirDataSource for AirDataInertialReferenceSystem {
     fn computed_airspeed(&self, adiru_number: usize) -> Arinc429Word<Velocity> {
         self.adirus[adiru_number - 1].computed_airspeed()
     }
 
     fn alpha(&self, adiru_number: usize) -> Arinc429Word<Angle> {
         self.adirus[adiru_number - 1].alpha()
+
+impl AdirsMeasurementOutputs for AirDataInertialReferenceSystem {
+    fn is_fully_aligned(&self, adiru_number: usize) -> bool {
```

Comment:
This signal doesn't exist in this form on the real thing, so probably not the right way to go. Instead the discrete words are sent on the bus, by the ADR and the IR.